src/modules/users/users.controller.ts

```diff
@@ -104,6 +104,39 @@ export default class UserController extends Api {
     }
   };
 
+  public createClub = async (
+    req: Request,
+    res: CustomResponse<Clubs>,
+    next: NextFunction
+  ) => {
+    try {
+      const club = await this.userService.createClub(
+        req.body,
+        req.user as JwtPayload
```

Comment:
this caused the forbidden error. You are not authenticated with bearer auth yet so this return undefined

Reply:
This is still the case as of now. We are troubleshooting why the user is undefined when creating a club. It might help if we could find what the real cause is here.